org.sonarlint.eclipse.core/META-INF/MANIFEST.MF

```diff
@@ -34,6 +33,7 @@ Require-Bundle: org.jsr-305;resolution:=optional,
  org.eclipse.core.filesystem,
  org.eclipse.team.core,
  org.sonarsource.sonarlint.core.sonarlint-client-api;bundle-version="4.0.0",
- org.sonarsource.sonarlint.core.sonarlint-core;bundle-version="4.0.0"
+ org.sonarsource.sonarlint.core.sonarlint-core;bundle-version="4.0.0",
+ org.eclipse.jdt.annotation;resolution:=optional
 Bundle-ActivationPolicy: lazy
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-RequiredExecutionEnvironment: JavaSE-11
```

Comment:
I believe this is a bit premature :smile: or did I miss something?

Reply:
Good catch. One remain from my (too) numerous tries...